First_project/First_project/Profile/LoginViewController.swift

```diff
@@ -0,0 +1,218 @@
+            scrollView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            scrollView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            
+            areaView.topAnchor.constraint(equalTo: scrollView.topAnchor),
+            areaView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor),
+            areaView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor),
+            areaView.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor),
+            areaView.widthAnchor.constraint(equalTo: scrollView.widthAnchor),
+            
+            logo.heightAnchor.constraint(equalToConstant: 100),
+            logo.widthAnchor.constraint(equalToConstant: 100),
+            logo.centerXAnchor.constraint(equalTo: areaView.centerXAnchor),
+            logo.topAnchor.constraint(equalTo: areaView.topAnchor,constant: 120),
+            
+            profileLogin.heightAnchor.constraint(equalToConstant: 50),
+            profileLogin.leadingAnchor.constraint(equalTo: areaView.leadingAnchor,constant: AppConstants.leftAnchorSize),
+            profileLogin.trailingAnchor.constraint(equalTo: areaView.trailingAnchor,constant: AppConstants.rightAnchorSize),
+            profileLogin.topAnchor.constraint(equalTo: logo.bottomAnchor,constant: 120),
+            
+            profilePassword.heightAnchor.constraint(equalToConstant: 50),
+            profilePassword.leadingAnchor.constraint(equalTo: areaView.leadingAnchor,constant: AppConstants.leftAnchorSize),
+            profilePassword.trailingAnchor.constraint(equalTo: areaView.trailingAnchor,constant: AppConstants.rightAnchorSize),
+            profilePassword.topAnchor.constraint(equalTo: profileLogin.bottomAnchor),
+            
+            loginButton.topAnchor.constraint(equalTo: profilePassword.bottomAnchor, constant: AppConstants.leftAnchorSize),
+            loginButton.leadingAnchor.constraint(equalTo: areaView.leadingAnchor, constant: AppConstants.leftAnchorSize),
+            loginButton.trailingAnchor.constraint(equalTo: areaView.trailingAnchor, constant: AppConstants.rightAnchorSize),
+            loginButton.heightAnchor.constraint(equalToConstant: 50),
+            loginButton.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor)
+        ])
+        
+        profileLogin.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
+        profilePassword.layer.maskedCorners = [.layerMaxXMaxYCorner, .layerMinXMaxYCorner]
+        
+        navigationController?.setNavigationBarHidden(true, animated: true)
+        
+    }
+    
+    // MARK Keyboard observers
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
```

Comment:
NotificationCenter.default.removeObserver(self)

This will remove all observers from class